decompile/preprocessing/preprocess.py

```diff
@@ -2,45 +2,70 @@
 import os
 import subprocess
 import shutil
+from multiprocessing import Pool
 
-C_CODE_FOLDER = "D:\Work\Research paper\DatasetC"
-ASSEMBLY_FOLDER = "D:\Work\Research paper\DatasetAO3"
-SOURCE_FOLDER = "D:\Work\Research paper\AnghaBench-master"
+folder_path = "/home/mohanned/Work/DataC"
+output_folder = "/home/mohanned/Work/Binary"
 
+def compile_to_binary(file_path: str) -> None:
+    """
+        Converting C code files into binary files
+        
+        Args:
+        file_path (str): It's the path for the C code file being compiled
+    """
+    binary_file = os.path.splitext(file_path)[0]
+    binary_file = binary_file[::-1]
+    EDIT = ""
+    for c in binary_file:
+        if(c == '/'):
+            break
+        EDIT += c
+    EDIT = EDIT[::-1]
+    out_file = output_folder + "/" + EDIT + ".out"
 
-def compile_to_assembly(c_code_file: str, output_file: str) -> bool:
-    """Compile C code to assembly.
+    assemble_command = f"gcc -c {file_path} -o {out_file}"
 
-    Args:
-        c_code_file_path (str): Path to C code file
-        output_file_path (str): Path to C code file
+    try:
+        subprocess.run(assemble_command, check=True, shell=True)
+        print(f"Assembly successful. Binary machine code saved as {out_file}")

```

Comment:
Remove logging for successful sub process. Only include logging for failed ones.